test/unit/lib/plugins/aws/package/compile/events/websockets/lib/stage.test.js

```diff
@@ -149,6 +149,29 @@ describe('#compileStage()', () => {
       });
     });
 
+    it('should set a DataProtectionPolicy in a Log Group if provider has logDataProtectionPolicy', () => {
+      const policy = {
+        Name: 'data-protection-policy',
+        Version: '2021-06-01',
+        Statement: [],
+      };
+      awsCompileWebsocketsEvents.serverless.service.provider.logDataProtectionPolicy = policy;
+
+      return awsCompileWebsocketsEvents.compileStage().then(() => {
+        const resources =
+          awsCompileWebsocketsEvents.serverless.service.provider.compiledCloudFormationTemplate
+            .Resources;
+
+        expect(resources[logGroupLogicalId]).to.deep.equal({
+          Type: 'AWS::Logs::LogGroup',
+          Properties: {
+            LogGroupName: '/aws/websocket/my-service-dev',
+            DataProtectionPolicy: policy,
+          },
+        });
+      });
+    });
```

Comment:
Let's configure all tests with `runServerless` (see last `describe` section in this file, it should be put there)

Reply:
Anything else missing? @medikoo

Reply:
@timo92 if everything is addressed and you feel it's ready for re-review, please re-request it in top right reviewers box. Thank you!